images/Makefile

```diff
@@ -26,17 +26,11 @@ apt-cacher.%: base.%
 base.%:
 	@$(MAKE) -C base PLATFORM=$*
 
-ceph.%: cross-gnu.linux_amd64
```

Comment:
i understand why the `linux_arm` stuff has been removed in this PR, but i'm not sure about these lines that are referencing `linux_amd64`.  Perhaps because this is for cross compilation, which also isn't needed because we are not building ceph from source anymore?

Reply:
we no longer have a base ceph image. we just pull the ceph packages into the rook image